local/db/activity_program.sql

```diff
@@ -0,0 +1,39 @@
+drop table IF EXISTS activity_program;
+
+create table activity_programs (
+  id                 serial primary key,
```

Comment:
PKここで定義ではなく、一番下でまとめちゃってください。
```suggestion
  id                 serial,
```